nexus/reconfigurator/execution/src/lib.rs

```diff
@@ -123,143 +131,512 @@ pub async fn realize_blueprint_with_overrides(
         "blueprint_id" => %blueprint.id
     );
 
-    datastore
-        .blueprint_ensure_external_networking_resources(&opctx, blueprint)
-        .await
-        .map_err(|err| {
-            vec![anyhow!(err).context(
-                "failed to ensure external networking resources in database",
-            )]
-        })?;
-
-    let sleds_by_id: BTreeMap<SledUuid, _> = datastore
-        .sled_list_all_batched(&opctx, SledFilter::InService)
-        .await
-        .context("listing all sleds")
-        .map_err(|e| vec![e])?
-        .into_iter()
-        .map(|db_sled| {
-            (SledUuid::from_untyped_uuid(db_sled.id()), Sled::from(db_sled))
-        })
-        .collect();
-
-    let deploy_disks_done = omicron_physical_disks::deploy_disks(
+    // Large enough to handle all the messages.
```

Comment:
Is this comment in the wrong place?

Reply:
Yeah, moved it.

Reply:
I made a common `update_engine::channel` function so callers don't have to determine a good channel size each time.